### DISCUSSION TITLE
Dark Theme not working / option not shown

### DISCUSSION THREAD

**Discussion by King-0f-Hero** [AppGround-io/bluetooth-keyboard-and-mouse-support#274] [Bugs & Problems]
I'm on a Pixel 6 Pro and I downloaded the app through the Play Store, but I figured I would get a better and faster response here compared to leaving a review or something like that.
My problem is that I'm not able to find an option for a dark theme even though there are screenshots of one in the Play Store.
Can somebody help me out, please?

**Comment by AdamKane41** [comment]
It should be in the App's Display settings, there's a couple of different themes, including dark ones... 🙂

**Reply by King-0f-Hero** [reply]
I was only in my Custom layouts expecting to have the settings in the side menu and didn't know you have to go into the "Mouse / Keyboard" layout to change the theme.
So now I managed to switch to dark mode with a theme color.
So as a suggestion to the developers: I think it would be more intuitive to have a button to global settings in the side menu and not hidden inside a specific layout's settings.
Otherwise I want to give props to the developers for an overall very well designed and functioning app. 👏
